### ISSUE TITLE
sendmail: can't connect to remote host (127.0.0.1): Connection refused

### ISSUE THREAD

**Issue by jaswinder97** [TheMoroccan09/alpine-sendmail#2]
@TheMoroccan09  do we need to expose any port for a docker container?

**Comment by TheMoroccan09** [comment]
> @TheMoroccan09 do we need to expose any port for a docker container?

no, but you should use the docker host and not the host one (127.0.0.1)
